icu4j/main/classes/core/src/com/ibm/icu/util/JapaneseCalendar.java

```diff
@@ -386,6 +386,12 @@ protected void handleComputeFields(int julianDay) {
      */
     static public final int HEISEI;
 
+    /**
+     * Constant for the era starting on May. 1, 2019 AD.
```

Comment:
Could you please remove the dot after May and git commit --amend ?

Reply:
Removed dot after May, and squashed.